syz-manager/manager.go

```diff
@@ -654,9 +655,45 @@ func (mgr *Manager) loadCorpus() []fuzzer.Candidate {
 		candidates = append(candidates, item)
 	}
 	log.Logf(0, "%-24v: %v (%v seeds)", "corpus", len(candidates), seeds)
+	// Let's favorize smaller programs, otherwise the poorly minimized ones may overshadow the rest.
+	sort.SliceStable(candidates, func(i, j int) bool {
+		return len(candidates[i].Prog.Calls) < len(candidates[j].Prog.Calls)
+	})
+	reminimizeSubset(candidates)
 	return candidates
 }
 
+// reminimizeSubset clears the fuzzer.ProgMinimized flag of a small subset of seeds.
+// The ultimate objective is to gradually clean up the poorly minimized corpus programs.
+// reminimizeSubset assumes that candidates are sorted in the order of ascending len(Prog.Calls).
+func reminimizeSubset(candidates []fuzzer.Candidate) {
+	if len(candidates) < 100 {
+		return
```

Comment:
Why?

Reply:
If we aim at <= 1%, then it doesn't make much sense to run the code below when there are fewer than 100 programs.

If not 100, there must still be some positive number -- we want to look at the top 10% biggest programs, so the candidates array must not be empty.

Reply:
> must not be empty
"not empty" looks like a reasonable condition.
You do `max(1, len(candidates)/100)`, so any corpus size >0 should be fine.